client/utilities/supportPricing/supporterPlusPricing.ts

```diff
@@ -51,27 +50,17 @@ export function getBenefitsThreshold(
 	return region[billingPeriod].minAmount;
 }
 
-export function getSuggestedAmounts(
-	currentAmount: number,
-	monthlyOrAnnual: 'Monthly' | 'Annual',
-) {
-	return monthlyOrAnnual === 'Monthly'
-		? suggestedAmountsMonthly(currentAmount)
-		: suggestedAmountsAnnual(currentAmount);
-}
-
 export function getSuggestedAmountsFromMainPlan(
 	mainPlan: PaidSubscriptionPlan,
 ) {
 	const currentAmount = mainPlan.price / 100;
-	const monthlyOrAnnual = calculateMonthlyOrAnnualFromBillingPeriod(
-		mainPlan.billingPeriod,
-	);
 
-	return getSuggestedAmounts(currentAmount, monthlyOrAnnual);
+	return mainPlan.billingPeriod === 'month'
+		? getSuggestedAmountsMonthly(currentAmount)
+		: getSuggestedAmountsAnnual(currentAmount);
 }
 
-function suggestedAmountsMonthly(currentAmount: number) {
+export function getSuggestedAmountsMonthly(currentAmount: number) {
```

Comment:
What are these calculations used for compared to the new ones?

Reply:
the suggested amounts when you update the amount of a supporter plus
<img width="890" alt="image" src="https://github.com/guardian/manage-frontend/assets/114918544/8db4c7bf-c42b-4fe6-99bc-d656d31e4c26">
confusing right? they're also the default fallback if someone goes over the range defined in the spreadsheet

Reply:
suggestions appreciated for how to name this in a way that makes it less confusing

Reply:
So one is for RC upsell and the other is for S+? If so maybe put the product name in the function name

Reply:
That's much clearer to me 👍